sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ResourceConfiguration.java

```diff
@@ -113,7 +119,16 @@ static Resource configureResource(
 
   // visible for testing
   static Resource filterAttributes(Resource resource, ConfigProperties configProperties) {
-    Set<String> disabledKeys = new HashSet<>(configProperties.getList(DISABLED_ATTRIBUTE_KEYS));
+    List<String> disabledAttibuteKeys = configProperties.getList(DISABLED_ATTRIBUTE_KEYS);
+    // TODO: Remove this once the deprecated property is removed.
+    if (disabledAttibuteKeys.isEmpty()) {
+      disabledAttibuteKeys = configProperties.getList(EXPERIMENTAL_DISABLED_ATTRIBUTE_KEYS);
+      if (!disabledAttibuteKeys.isEmpty()) {
+        logger.warning(
+            "otel.experimental.resource.disabled.keys is deprecated. Please use otel.resource.disabled.keys instead.");
```

Comment:
How about we warn users that it will be removed after the 1.45.0 release, which is 3 releases?
```suggestion
            "otel.experimental.resource.disabled.keys is deprecated and will be removed after 1.45.0 release. Please use otel.resource.disabled.keys instead.");
```

Reply:
Thanks for adding perspective. This looks a lot better and I added this to warning log.